homeassistant/components/plex/config_flow.py

```diff
@@ -44,11 +55,14 @@ class PlexFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
     def __init__(self):
         """Initialize the Plex flow."""
         self.current_login = {}
+        self.discovery_info = {}
         self.available_servers = None
 
     async def async_step_user(self, user_input=None):
         """Handle a flow initialized by the user."""
         if user_input is not None:
+            if user_input["manual_setup"]:
+                return await self.async_step_manual_setup()
```

Comment:
Should we pass the token to the manual step? That way the user can fill in the token in the first step and continue filling in the rest in the next step. We can populate the defaults with the existing input of present.

Reply:
Definitely. Good call.